### PR TITLE
Add Lisa keyboard

### PR DESCRIPTION
**Pull request by key10iq**
## Description

Lisa is a QMK and VIA compatible Tadpole mount keyboard with Prime_E layout.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* Added new keyboard - keyten/lisa (QMK and VIA)

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by key10iq** [comment]
I don't understand what formatting is required of me

image

**Comment by sigprof** [comment]
As usual, lint wants Unix-style line endings in all files, and you have CRLF in `info.json`.

**Comment by key10iq** [comment]
> As usual, lint wants Unix-style line endings in all files, and you have CRLF in `info.json`.

Spasibo, Sergey

File: keyboards/keyten/lisa/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS,          KC_TRNS,          KC_TRNS, KC_TRNS,          KC_TRNS,                   KC_TRNS, KC_TRNS
+	)
+};
+
+void matrix_init_user(void) {
+    setPinOutput(B2);
+//    writePinLow(B2);
+    writePinHigh(B2);
+}
```
Comment (waffle87) [inline keyboards/keyten/lisa/keymaps/default/keymap.c:33]: This should be moved to the keyboard level in a `lisa.c` file.
```suggestion
```

File: keyboards/keyten/lisa/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+//		    break;
+//        case 3:
+//		    writePinHigh(B2);
+//		    break;	
+//        default:
+//		    writePinLow(B2);
+//        break;
+//    }
+//    return state;
+//}
```
Comment (waffle87) [inline keyboards/keyten/lisa/keymaps/default/keymap.c:51]:
```suggestion
```

File: keyboards/keyten/lisa/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,51 @@
+/* SPDX-License-Identifier: GPL-2.0-or-later */
```
Comment (waffle87) [inline keyboards/keyten/lisa/keymaps/default/keymap.c:1]:
```suggestion
// Copyright 2024 QMK
// SPDX-License-Identifier: GPL-2.0-or-later
```

File: keyboards/keyten/lisa/keymaps/via/keymap.c [hunk 4]
```diff
@@ -0,0 +1,51 @@
+/* SPDX-License-Identifier: GPL-2.0-or-later */
```
Comment (waffle87) [inline keyboards/keyten/lisa/keymaps/via/keymap.c:1]:
```suggestion
// Copyright 2024 QMK
// SPDX-License-Identifier: GPL-2.0-or-later
```

File: keyboards/keyten/lisa/keymaps/via/keymap.c [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+//		    break;
+//        case 3:
+//		    writePinHigh(B2);
+//		    break;	
+//        default:
+//		    writePinLow(B2);
+//        break;
+//    }
+//    return state;
+//}
```
Comment (waffle87) [inline keyboards/keyten/lisa/keymaps/via/keymap.c:51]: As above.
```suggestion
```

**Review by waffle87 — CHANGES_REQUESTED** [review]

File: keyboards/keyten/lisa/lisa.c [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+//		    break;
+//        case 3:
+//		    writePinHigh(B2);
+//		    break;	
+//        default:
+//		    writePinLow(B2);
+//        break;
+//    }
+//    return state;
+//}
```
Comment (waffle87) [inline keyboards/keyten/lisa/lisa.c:28]:
```suggestion
void matrix_init_kb(void) {
    gpio_set_pin_output(B2);
    gpio_write_pin_high(B2);
    matrix_init_user();
}
```

File: keyboards/keyten/lisa/lisa.c [hunk 7]
```diff
@@ -0,0 +1,10 @@
+// Copyright 2024 QMK
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "quantum.h"
+
+void matrix_init_kb(void) {
+    gpio_set_pin_output(B2);
+    gpio_write_pin_high(B2);
```
Comment (fauxpark) [inline keyboards/keyten/lisa/lisa.c:8]: What is this for?

Comment (key10iq) [reply keyboards/keyten/lisa/lisa.c:8]: One LED indicator is always on

Comment (fauxpark) [inline keyboards/keyten/lisa/lisa.c:8]:
```suggestion
#include "quantum.h"

#define LED_INDICATOR_PIN B2

void matrix_init_kb(void) {
    gpio_set_pin_output(LED_INDICATOR_PIN);
    gpio_write_pin_high(LED_INDICATOR_PIN);
```
Feel free to rename this to something more descriptive.

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Review by waffle87 — APPROVED** [review]

**Pull request merged**